docs/console/console-extensibility/registration.md

```diff
@@ -34,85 +37,169 @@ The route to contact is `PUT /api/extensibility/tenants/{tenantId}/extensions`,
+You can find the icons at this [link](https://react-icons.github.io/react-icons/search/), but only Ant, Feather and Phosphor icons are supported. After pick an icon, you can use the React component name represented in the code example (e.g. the Phosphor icon `<PiAirplaneTiltFill />` has `PiAirplaneTiltFill` as name).
+
+
+## Get registered Extensions by list
+
+The route `GET /api/extensibility/tenants/{tenantId}/extensions` allows you to retrieve all registered extensions under a specific Company. This functionality is readily accessible via miactl `extensions list` command or via the API Portal under the 'Companies' or 'Extensibility' tags, provided you have the Company Owner role for the selected Company.
 
 **Path Params**
 - `tenantId`: insert the tenant ID of the Company which you want to retrieve the registered extensions of
 
+**Query Params**
+- `resolveDetails`: returns the extension list resolving all the details (`permissions`, `visibilities`, `menu` and `category.`) if defined
+
 **Response on success**
-Array of registered extensions metadata:
+Array of registered extensions info:
+
+```json
+{
+    "extensionId": "string",
+    "name": "string",
+    "activationContexts": "Array<string>",
+    "description": "string",
+    "entry": "string",
+    "type": "string",
+    "destination": {
+        "id": "string",
+        "path": "string",
+    },
+    "iconName": "string", 
+    "permissions": "Array<string>",
+    "visibilities": "Array<{contextType, contextId}>",
+    "menu": {
+        "id": "string",
+        "labelIntl": {"en": "string", "it": "string"},
+        "order": "number",
+    },
+    "category": {
+        "id": "string",
+        "labelIntl": {"en": "string", "it": "string"},
+        "order": "number",
+    }
+}
+```
 
-- `extensionId`: extension identifier
-- `name`: extension name
-- `description`: extension description
+The `visibilities` key in the response in which specific context the extension is active, instead the other ones are the same info applied when the extension is been registered. This keys are explained above in the [how to register your extension](#how-to-register-your-extension) paragraph.
+
+:::tip
+**[See the example in the tutorial page](/console/tutorials/create-extension.md#2-check-that-the-new-extension-is-registered)**
+:::
+
+## Get registered Extension by ID
+
+You can retrieve also the info of a specific registered extension using the route `GET /api/extensibility/tenants/{tenantId}/extensions/:extensionId`. This API is accessible via miactl `extensions get` command or via the API Portal under the 'Companies' or 'Extensibility' tags, provided you have the Company Owner role for the selected Company.
+
+**Path Params**
+- `tenantId`: insert the tenant ID of the Company which you want to retrieve the registered extensions of
+- `extensionId`: insert the extension ID of the desired extension
+
+**Response on success**
+Array of registered extensions info:
+
+```json
```

Comment:
can you put this in a collapsed panel so that it does not take too much space in the page?